config/production.json

```diff
@@ -69,6 +69,7 @@
 		"plans/personal-plan": true,
 		"post-editor/author-selector": true,
 		"post-editor/insert-menu": true,
+		"post-editor/track-paste-events": true,
```

Comment:
To clarify on the above: still not sure how the clipboard API will play with an electron-based app, on several OS, so I've just prepared the code for the case we need to disable that.